### DISCUSSION TITLE
Activating Phosphorus response for soybean in APSIMx

### DISCUSSION THREAD

**Discussion by pmasikati** [APSIMInitiative/ApsimX#9232] [Q&A]
Greetings, am doing simulations for soybean production in southern Africa  (Malawi, Zambia and Mozambique) and want to assess P response in soybean. P is a limiting factor in soybean and other legume production in the region. Can someone assist with information on how I can do this in APSIMx.

**Comment by hol353** [comment]
There is a soil phosphorous model in Next Gen but currently none of the plant models in APSIM respond to phosphorus. It is on our very long todo list but won't happen soon.

**Reply by pmasikati** [reply]
Thanks for the response Dean, looking forward to the happenings on P in the future.

**Reply by ysharaby1102** [reply]
Is there still no phosphorus response in the Soybean model?
I have recently tried to simulate fertilized soybean in Illinois via APSIM NG, as the P-fertilization had no effect on yield or any other biomass/architecture feature...

**Comment by hut104** [comment]
There is no P response in Soybean, or any other model in APSIM.  A prototype soil P model has been tested for incubation studies.  We currently have no test data sets for P response in soybean or other crops with which to develop or test new capabilities here.  Hopefully we may be able to find funding to drive improvements in this space.
________________________________

**Reply by lpmorenoc** [reply]
It has been almost one year, but I was wondering if the inclusion of phosphorus is still in the long wait list
